relayer/pkg/chainlink/txm/txm.go

```diff
@@ -101,7 +100,11 @@ func (txm *starktxm) run() {
 
 			// fetch client if needed
 			if txm.client == nil {
-				txm.client = txm.getClient()
+				var err error
+				txm.client, err = txm.getClient()
+				if err != nil {
+					txm.lggr.Errorw("unable to fetch client", "error", err)
```

Comment:
Should this `continue` the for loop to try again on next tick, rather than moving on with `txm.client == nil`?

Reply:
Deleted my other comment to group it with a similar one here, basically "How do we plan to recover from this?" is the Q.

Reply:
yeah there should be a `continue` with that...I'll fix that in a follow up PR with the DB fix